### ISSUE TITLE
MetricsTests fails because of flush changes

### ISSUE THREAD

**Issue by friedbunny** [mapbox/mapbox-gl-native#1616]
```objc
Test Suite 'MetricsTests' started at 2015-05-21 19:51:07 +0000
2015-05-21 12:51:13.553 Mapbox GL Tests[24024:3229395] -[MGLMapboxEvents flushAfter]: unrecognized selector sent to instance 0x7fdd4053b820
<unknown>:0: error: MetricsTests : -[MGLMapboxEvents flushAfter]: unrecognized selector sent to instance 0x7fdd4053b820
(
	0   CoreFoundation                      0x0000000110155c65 __exceptionPreprocess + 165
	1   libobjc.A.dylib                     0x000000010fdeebb7 objc_exception_throw + 45
	2   CoreFoundation                      0x000000011015d0ad -[NSObject(NSObject) doesNotRecognizeSelector:] + 205
	3   CoreFoundation                      0x00000001100b313c ___forwarding___ + 988
	4   CoreFoundation                      0x00000001100b2cd8 _CF_forwarding_prep_0 + 120
	5   Test Bundle                         0x000000011a091f2e -[MetricsTests beforeAll] + 174
	6   Test Bundle                         0x000000011a0af831 +[KIFTestCase performSetupTearDownWithSelector:] + 80
    ...
)
Test Suite 'MetricsTests' failed at 2015-05-21 19:51:13 +0000.
	 Executed 0 tests, with 1 failure (1 unexpected) in 0.000 (5.599) seconds
```

**Comment by 1ec5** [comment]
I turned `MGLMapboxEvents.flushAfter` into a constant in #1491, but the tests in #1517 were never updated to work with #1491.

@incanus, it sounds like this issue will block #1452.

**Comment by jfirebaugh** [comment]
Fixed by @friedbunny in 10990475b9a72ba07f36058e8d3c04e42e1f5c70.
